docs/howto/command_line.rst

```diff
@@ -36,3 +36,35 @@ using:
 .. code-block:: console
 
     $ export PROCRASTINATE_APP=dotted.path.to.app worker
+
+As a general rule, all parameters have an environment variable equivalent, named
+``PROCRASTINATE_SOMETHING`` where ``SOMETHING`` is the uppercased long name of the
+option, with ``-`` replaced with ``_``.
+
+Logging
+^^^^^^^
+
+Three different options allow you to control how the command-line tool should log
+events:
+
+- Verbosity controls the log level (you'll see message of this level and above):
+
+  +-------+---------------------------+-------------+
+  | Flags | Environment equivalent    | Log level   |
+  +=======+===========================+=============+
+  |       | PROCRASTINATE_VERBOSITY=0 | ``warning`` |
+  +-------+---------------------------+-------------+
+  | -v    | PROCRASTINATE_VERBOSITY=1 | ``info``    |
+  +-------+---------------------------+-------------+
+  | -vv   | PROCRASTINATE_VERBOSITY=2 | ``debug``   |
+  +-------+---------------------------+-------------+
+
+- Log format: ``--log-format=`` / ``PROCRASTINATE_LOG_FORMAT=`` lets you control how
+  the log line will be formatted. It uses ``%``-style placeholders.
```

Comment:
Do you want to append "by default" to this sentence, as the format style may be changed using the `--log-format-style` flag?

Reply:
```suggestion
  the log line will be formatted. It uses ``%``-style placeholders by default.
```